.github/workflows/tests.yml

```diff
@@ -24,8 +24,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install Dependencies
       run: |
-        apt get update
-        apt install make
+        sudo apt install make
```

Comment:
I'm fine with this change, but note that in some systems that are very "minimal", you cannot install before update (it doesn't have the packages list, will not find `make`)

Reply:
Thanks! I added the update line. Also, thanks for the review!